cmd/executor/cmd/root.go

```diff
@@ -113,6 +113,11 @@ func checkContained() bool {
 
 func checkDockerfilePath() error {
 	if util.FilepathExists(dockerfilePath) {
+		abs, err := filepath.Abs(dockerfilePath)
+		if err != nil {
+			return err
+		}
+		dockerfilePath = abs
```

Comment:
Hmm, could we refactor this a bit to avoid mutating the global variable in here?

Reply:
done